webpack/stories/docs/internationalization.md

```diff
@@ -1,14 +1,42 @@
 # Translations
 
-All strings need to be translated. We use gettext's underscore `__()` function as you know it from our asset pipeline code.
+All strings need to be translated.
```

Comment:
This is a nice intro paragraph indeed @boaz1337. Since this PR has been open for more than two weeks already I don't won't block it with this. May I ask you to open your PR with this language adjustment?